doc/installing-an-operating-system.md

```diff
@@ -3,6 +3,10 @@ Installing an Operating System
 
 The general idea is to follow the generic UEFI instructions for ARM or AArch64.
 
+Tow-Boot cannot write to EFI variables. Your operating system's documentation
```

Comment:
(Note to self) Once the operating system is booted, writing to EFI variables will not be possible.